spark-cassandra-connector-demos/simple-demos/src/main/scala/com/datastax/spark/connector/demo/BasicWriteDemo.scala

```diff
@@ -0,0 +1,21 @@
+package com.datastax.spark.connector.demo
+
+import org.apache.spark.{SparkContext, SparkConf}
+
+import com.datastax.spark.connector._
+
+object BasicWriteDemo extends App {
+
+  val conf = new SparkConf(true)
+    .setAppName("Write test")
+    .set("spark.cassandra.connection.host", "localhost")
+    .set("spark.cassandra.output.concurrent.writes", "1")
+    .set("spark.cassandra.output.batch.level", "replica_set")
+    .setMaster("local[1]")
+
+  val sc = new SparkContext(conf)
+
+  val col = for (i <- 1 to 1000000) yield (i, "foo" + i)
+  val rdd = sc.parallelize(col, 20)
+  rdd.saveToCassandra("test", "kv")
```

Comment:
Hmm, I actually never wanted to commit this file. But indeed, let's create a table beforehand.